model-optimizer/mo/back/offline_transformations.py

```diff
@@ -3,28 +3,48 @@
 
 import argparse
 
-if __name__ == "__main__":
-    parser = argparse.ArgumentParser()
-    parser.add_argument("--input_model")
-    parser.add_argument("--framework")
-    args = parser.parse_args()
-    path_to_model = args.input_model
+from mo.utils.error import Error
+from mo.utils.cli_parser import parse_transform
 
-    # This variable is only needed by GenerateMappingFile transformation
-    # to produce correct mapping
-    extract_names = True if args.framework in ['tf', 'mxnet', 'kaldi'] else False
 
+def get_available_transformations():
     try:
-        from openvino.inference_engine import IECore, read_network # pylint: disable=import-error
-        from openvino.offline_transformations import ApplyMOCTransformations, GenerateMappingFile, CheckAPI # pylint: disable=import-error
+        from openvino.offline_transformations import ApplyLowLatencyTransformation # pylint: disable=import-error
+        return {
+            'LowLatency': ApplyLowLatencyTransformation,
+        }
     except Exception as e:
-        print("[ WARNING ] {}".format(e))
-        exit(1)
+        return {}
+
+
+def apply_offline_transformations(input_model: str, framework: str, transforms: list):
+    # This variable is only needed by GenerateMappingFile transformation
+    # to produce correct mapping
+    extract_names = True if framework in ['tf', 'mxnet', 'kaldi'] else False
 
-    CheckAPI()
+    from openvino.inference_engine import read_network # pylint: disable=import-error
+    from openvino.offline_transformations import ApplyMOCTransformations, GenerateMappingFile # pylint: disable=import-error
 
-    net = read_network(path_to_model + "_tmp.xml", path_to_model + "_tmp.bin")
-    net.serialize(path_to_model + ".xml", path_to_model + ".bin")
-    path_to_mapping = path_to_model + ".mapping"
+    net = read_network(input_model + "_tmp.xml", input_model + "_tmp.bin")
+
+    available_transformations = get_available_transformations()
+
+    for name, args in transforms:
+        if name not in available_transformations.keys():
+            raise Error("Transformation {} is not available.".format(name))
```

Comment:
Is it possible to show list of available transformations when incorrect is specified? E.g. when we specify incorrect --framework - MO will output list of available frameworks